packages/cli/test/test-utils.js

```diff
@@ -2,9 +2,9 @@
 // Node module: @loopback/cli
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
-
 'use strict';
 
+const fsextra = require('fs-extra');
```

Comment:
AFAIK, `fsextra` is exposing all `fs` APIs and adds some new ones. There is no need to have both `fs` and `fsextra` variables. Here is what we usually do:

```js
- const fs = require('fs');
+ const fs = require('fs-extra');
```

Reply:
> AFAIK, `fsextra` is exposing all `fs` APIs and adds some new ones. There is no need to have both `fs` and `fsextra` variables.

@marvinirwin PTAL ☝️

Reply:
woops, fixed